.github/workflows/changelog_generation.yaml

```diff
@@ -40,7 +39,7 @@ jobs:
         python3 .github/release-note-generation/split_release_note.py pr_body.txt .
     - name: Commit the change and push to the pull request branch
       run: |
-        git config user.email "yoshi-code-bot[bot]@users.noreply.github.com"
+        git config user.email "yoshi-code-bot@users.noreply.github.com"
```

Comment:
This is another fix on this pull request. It was failing CLA check in https://github.com/googleapis/google-cloud-java/commit/a3412ef0fbfe1775a36c34791209db813d30030f. I confirmed this email address passes the CLA check https://github.com/googleapis/google-cloud-java/commit/e539052d0920b591fbbe070d767e77352fe4d6c1.